styles.css

```diff
@@ -0,0 +1,106 @@
+*{
```

Comment:
please add the indentation and whitespaces according to the linters so that you won't have any linter errors as the code looks more clean and professional.